<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Portfolio</title>
  <meta name="description" content="Welcome to Antonio Ortega's personal website! Here you can explore his portfolio and his profile.">

  <link rel="stylesheet" href="/css/main.css" type="text/css">
  <link rel="stylesheet" href="/blogposts.css" type="text/css">

  <link rel="canonical" href="http://localhost:4000/portfolio/">
  
  
  <link rel="alternate" hreflang="dk" href="/pages/projects/dk.html" />
  <link rel="alternate" hreflang="en" href="/portfolio/" />
  <link rel="alternate" hreflang="es" href="/pages/projects/es.html" />
  <link rel="alternate" type="application/rss+xml" title="Antonio Ortega" href="http://localhost:4000">
</head>
<body><header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/index.html">Antonio Ortega</a>

    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>

      <div class="trigger">
 <!--    -->
         

        
        
        
        
          
        
        
        
        
        
          
        
        
        
          
          <a class="page-link" href="/pages/cv/en.html">CV </a>
          
        
        
        
          
          <a class="page-link" href="/portfolio/">Portfolio </a>
          
        
        
        
          
          <a class="page-link" href="/posts/">Posts </a>
          
        
        
      </div>
    </nav>

  </div>

</header>

<div class="aParent page-content wrapper">

 <div class="wrapper" style="float: left">
   <img class = "logo" src="http://localhost:4000/assets/images/ku_logo.png" alt="KU logo" height="66px" width="66px">
 </div>

 <div class="wrapper" style="text-align: right; line-height: 2em; float: right">
   
   
 
   
   <!--  -->
    <a href="/pages/projects/dk.html" class="dk">



<img src="/assets/images/dannebrog.png" alt="" height="28.66px" width="44px">

</a>
 <a href="/portfolio/" class="en">

<img src="/assets/images/flag.gif" alt="" height="28.66px" width="44px">



</a>
 <a href="/pages/projects/es.html" class="es">


<img src="/assets/images/bandera.jpg" alt="" height="28.66px" width="44px">


</a>


   <!--  <a href="/pages/projects/dk.html" class="dk"> dk </a>  <a href="/portfolio/" class="en"> en </a>  <a href="/pages/projects/es.html" class="es"> es </a>  -->
 </div>

</div>


<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <header class="post-header">
    <h1 class="post-title">Portfolio</h1>
  </header>

  <div class="post-content">
    <style>
  .myspan {
    list-style-position: inside;
    margin-left: 2em;
  }
</style>

<h3 id="original-work">Original work</h3>

<p>These projects comprise either extended analysis of published data, or  completely independent work.</p>

<p><strong>2018</strong></p>

<ul>
  <li><a href="http://people.binf.ku.dk/rnq313/Datrik_test/report.pdf">Technical test for the Junior Data Scientist position at Datrik Intelligence”</a></li>
</ul>

<p>This report consisted of my deliverable for the technical test required by the Data Analytics company <a href="https://datrik.com/"><em>Datrik</em> <em>Intelligence</em></a> to access their position as Junior Data Scientist. Its CEO is a well-known data scientist who has been #1 in Kaggle worldwide. The test consisted of a Explorative Data Analysis (EDA) coupled with the development of two predictive models.</p>

<div class="container">
    <div class="row">

      <div class="span4 myspan">
          <p><strong>What I learned:</strong></p>
          <ul>
          <li>Multiple Correspondence Analysis (MCA) in FactoMiner</li>
          <li>Tree-based modelling in scikit-learn</li>
          </ul>
      </div><!--/span-->
  </div>
</div>
<div class="row spacer25"></div>

<ul>
  <li><a href="https://github.com/antortjim/Thesis-Code">Master Thesis titled “Development of label-free quantification methods in proteomics”.</a></li>
</ul>

<p>(I) Compilation of an open-source free (cost and license) pipeline for the analysis and quantification of DDA proteomics datasets in a GNU/Linux environment. Makes extensive use of the tools developed at the Compomics group at UGhent (Belgium).</p>

<p>(II) Writing of a statistical model for the probabilistic quantification of protein ratios between samples using the PyMC3 framework, focusing on the assessment of uncertainty. Carried out at Advanced analytics in Novozymes.</p>

<div class="container">
    <div class="row">

      <div class="span4 myspan">
          <p><strong>What I learned:</strong></p>
          <ul>
          <li>Probabilistic programming in PyMC3</li>
          <li>DDA Proteomics analysis</li>
          <li>Pipelining in bash</li>
          </ul>
      </div><!--/span-->
  </div>
</div>
<div class="row spacer25"></div>

<ul>
  <li><a href="http://people.binf.ku.dk/rnq313/binfProject2/report.pdf">Bioinformatics project titled “Classification and prognosis of Rheumatoid Arthritis patients. A case study”.</a>
Data cleaning and engineering (Scikit-learn), and development of a vanilla neural neutwork (TensorFlow) for classification of patients into 2 remission groups (progressor or not). An exploration of the architecture space revealed the dimensions of the best NN model and achieved a 0.78 AUC in the test set. Carried out at Nordic Bioscience.</li>
</ul>

<div class="container">
    <div class="row">

      <div class="span4 myspan">
          <p><strong>What I learned:</strong></p>
          <ul>
          <li>Data preprocessing in Python</li>
          <li>Vanilla NNs with TensorFlow</li>
          <li>Classifier evaluation with scikit-learn</li>
          </ul>
      </div><!--/span-->
</div>
</div>

<div class="row spacer25"></div>

<p><strong>2017</strong></p>

<ul>
  <li>
    <p><a href="http://people.binf.ku.dk/rnq313/binfProject1/report.pdf">Bioinformatics project titled “An interactive visualization tool for clinical hypothesis generation”.</a>
Development of a Shiny App for straightforward visualizations of health data. Powered by ggplot2, the app performs an exploratory analysis of the PERF database using boxplots, scatterplots, and a correlogram. The data can be filtered by different categories to interactively modify the graphics and download subsets of the databse. Carried out at Nordic Bioscience.</p>
  </li>
  <li>
    <p><a href="http://people.binf.ku.dk/rnq313/binfProject4/report.pdf">“Bioinformatics project titled “Finding patterns in fitness, nutrition and lifestyle-associated genomics data”.</a>
Analysis of SNP array datasets from a client database together with the 1000 genomes panel, drawing conclusions on the predictive power of genomic variants. I worked with PLINK, Bedtools, ADMIXTURE, SQL and ggplot2.</p>
  </li>
  <li>
    <p><a href="http://people.binf.ku.dk/rnq313/ATB/part2.html">Assessing admixture proportions.</a> Playing on top of the NGSadmix program output, a likelihood model is built to assess the significance of the found admixture contributions.</p>
  </li>
  <li>
    <p><a href="http://people.binf.ku.dk/rnq313/ATB/part1.html">EM Algorithm and NGS data.</a> An implementation of the Expectation Maximization algorithm applied to the estimation of base composition in haploid genomes.</p>
  </li>
  <li>
    <p><a href="http://people.binf.ku.dk/rnq313/report.pdf">Population Genetics final exam</a>. Working with SNP data from zebra species in the PLINK suite. The main topics focused on the genetic support for the distinction between plains subspecies and the position of the extinct <em>E. quagga quagga</em> in the phylogenetic tree of zebras.</p>
  </li>
  <li>
    <p><a href="https://antortjim.github.io/posts/2016/12/21/ioslides-and-Shiny-apps-showcase.html">Statistics for e-Science work</a>, a ioslides presentation and Shiny apps developed as part of the weekly exercises in this subject at KU.</p>
  </li>
  <li>
    <p><a href="http://people.binf.ku.dk/rnq313/rmsd_nmers/protein.pdf">Structural Bioinformatics final exam</a>. Students were asked to implement a Python pipeline to compute RMSD distributions on 5-mer peptides and interpret the results. Code  available at <a href="https://github.com/antortjim/structural_bioinformatics">Github</a>.</p>
  </li>
</ul>

<p><strong>2016</strong></p>

<ul>
  <li>
    <p><a href="http://people.binf.ku.dk/rnq313/EXAM/examen.pdf">Bioinformatics and Genomic Analysis final exam</a> consisting of 2 parts:
-Extended ChIP-seq analysis for prediction of PIF1 and PIF4 interaction in <em>A. thaliana</em>.
 -RNA-seq <em>de novo</em> analysis of early and late embrionary states in <em>X. tropicalis</em> using edgeR.</p>
  </li>
  <li>
    <p><a href="http://people.binf.ku.dk/rnq313/TFG/tfg.pdf">Bachelor Thesis</a>, entitled “Caracterización de la proteína All1873” (in Spanish).</p>
  </li>
</ul>

<h3 id="replicates-of-previously-published-scientific-works">Replicates of previously published scientific works</h3>

<p>The projects below are just replicates of the published computational analysis carried out by other scientists. Thus, despite there might be minor differences, the bulk of the creation is <em>not</em> my own original work. Reference to the scientific article projects are based on may be found in the <em>references</em> section.</p>

<div class="container">

    <div class="row">

      <div class="span4">
        <h4><a href="http://people.binf.ku.dk/rnq313/BMS_1/html"> Microarray data analysis</a></h4>
        <p> Data for 5 conditions in mice lung tissue </p>
        <p><strong>What I learned:</strong></p>
          <ul>
          <li>Bioconductor and R packages: limma, affy</li>
          <li>Quality control of samples</li>
          <li>Graphics for results interpretation: scatter and volcano plots, Venn diagrams, heatmaps</li>
          <li>Microarray tech pros and cons </li>
          <li>GEO database</li>
          </ul>
      </div><!--/span-->

      <div class="span4">
      <h4><a href="http://people.binf.ku.dk/rnq313/BMS_2/html"> RNA-seq data analysis</a></h4>
      <p> Data for 3 watering conditions in <em>A. thaliana</em> </p>
      <p><strong>What I learned:</strong></p>
        <ul>
          <li>Tuxedo protocol: bowtie, cufflinks, cuffmerge, cuffdiff and cummeRbund</li>
          <li>Working on a Sun Grid Engine cluster through the command line</li>
          <li>Gene set enrichment analysis (GSEA)</li>
          <li>Gene ontology (GO) </li> 
          <li>SRA database</li>
        </ul>
      </div><!--/span-->
    
    </div><!--/row-->

    <div class="row spacer25"></div>

    <div class="row">

      <div class="span4">
        <h4><a href="http://people.binf.ku.dk/rnq313/BMS_3/html"> Gene coexpression networks</a></h4>
        <p> Studying correlations in wine yeast gene expression </p>
        <p><strong>What I learned:</strong></p>
          <ul>
          <li>Network theory: scale-free and small-world networks</li>
          <li>Biological networks motifs: AR, DFL, IFFL and CFFL </li>
          <li>Cytoscape software and igraph package </li>
          <li>Correlation analysis </li>
          </ul>
      </div><!--/span-->

      <div class="span4">
      <h4><a href="http://people.binf.ku.dk/rnq313/BMS_4/html"> Cell physiology simulations </a></h4>
      <p> Simulation on Ca<sup>2+</sup> spontaneous peaks </p>
      <p><strong>What I learned:</strong></p>
        <ul>
          <li>Implement biological networks and motifs </li>
          <li>COPASI and Cell Designer software</li>
          <li>Effect of network motifs on its system's dynamics</li>
        </ul>
      </div><!--/span-->

    </div><!--/row-->

    <div class="row spacer25"></div>

    <div class="row">

      <div class="span4">
        <h4><a href="http://people.binf.ku.dk/rnq313/BAG_1/html"> RNA-seq and ChIP-seq integral analysis</a></h4>
        <p> Development of a workflow for data integral analysis. Gene Set Analysis was carried on subsets of interesting genes as found by RNA-seq and ChIP-seq </p>
        <p><strong>What I learned:</strong></p>
          <ul>
          <li>Shell scripting and blackboard messaging</li>
          <li>ChIP-seq analysis software: MACS and PeakAnalyzer </li>
          <li>Motif finders: HOMER</li>
          <li>KEGG Database </li>
          <li>ggplot2 and other "Hadley verse" R packages </li>
          </ul>
      </div><!--/span-->

      <div class="span4">
      <h4><a href="http://people.binf.ku.dk/rnq313/BAG_2/synteny.pdf"> Synteny study on Python </a></h4>
      <p> Analysis of <em>M. pneumoniae</em> and <em>M. genitalium</em> </p>
      <p><strong>What I learned:</strong></p>
        <ul>
          <li> Python scripting and Biopython module </li>
          Download script <a href="http://people.binf.ku.dk/rnq313/BAG_2/tarea_2.py"> here </a>
          <li> The time Linux utility</li>
        </ul>
      </div><!--/span-->

    </div><!--/row-->

    <div class="row spacer25"></div>


    <div class="row">

      <div class="span4">
        <h4><a href="http://people.binf.ku.dk/rnq313/BAG_3/tarea_3.html"> RNA-seq <em>de novo</em> analysis</a></h4>
        <p> <em>De novo</em> genome assembly on frog genome and differential expression analysis on 2 different development states </p>
        <p><strong>What I learned:</strong></p>
          <ul>
          <li> Genome assembly and assessment methods: Trinity </li>
          <li> Transdecoder </li>
          <li> KAAS </li>
          </ul>
      </div><!--/span-->

      <div class="span4">
      <h4><a href="http://people.binf.ku.dk/rnq313/BAG_4/tarea4.pdf"> Variant call analysis </a></h4>
      <p> Discovering differences between an arbitrary <em>A. thaliana</em> genome and a reference genome </p>
      <p><strong>What I learned:</strong></p>
        <ul>
          <li> bwa </li>
          <li> GATK software suite </li>
          Download shell script <a href="http://people.binf.ku.dk/rnq313/BAG_4/GWAS.sh"> here </a>
        </ul>
      </div><!--/span-->

    </div><!--/row-->

    <div class="row spacer25"></div>
</div>


  </div>

</article>

      </div>
    </main><footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Antonio Ortega</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>Antonio Ortega</li>
          <li><a href="mailto:antortjim1@gmail.com">antortjim1@gmail.com</a></li>
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/antortjim"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">antortjim</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/antortjim"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">antortjim</span></a>

          </li>
          
        </ul>
      </div>

      <div class="footer-col footer-col-3">
        <p>Welcome to Antonio Ortega's personal website! Here you can explore his portfolio and his profile.</p>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
