---
layout: page
title: Portfolio
permalink: /portfolio/
ref: portfolio
lang: en
weight: 2
---

### Original work ###

These projects comprise either extended analysis of published works, or  completely independent work.

* [Assessing admixture proportions.](http://people.binf.ku.dk/rnq313/ATB/part2.html) Playing on top of the NGSadmix program output, a likelihood model is built to assess the significance of the found admixture contributions.

* [EM Algorithm and NGS data.](http://people.binf.ku.dk/rnq313/ATB/part1.html) An implementation of the Expectation Maximization algorithm applied to the estimation of base composition in haploid genomes.


* [Bioinformatics and Genomic Analysis final exam](http://people.binf.ku.dk/rnq313/EXAM/examen.pdf) consisting of 2 parts:
 
   1. Extended ChIP-seq analysis for prediction of PIF1 and PIF4 interaction in *A. thaliana*
   2. RNA-seq *de novo* analysis of early and late embrionary states in *X. tropicalis* using edgeR.

* [Bachelor Thesis](http://people.binf.ku.dk/rnq313/TFG/tfg.pdf), entitled "Caracterización de la proteína All1873" (in Spanish).

* [Structural Bioinformatics final exam](http://people.binf.ku.dk/rnq313/rmsd_nmers/protein.pdf). Students were asked to implement a Python pipeline to compute RMSD distributions on 5-mer peptides and interpret the results. Code  available at [Github](https://github.com/antortjim/structural_bioinformatics).

* [Statistics for e-Science work](https://antortjim.github.io/posts/2016/12/21/ioslides-and-Shiny-apps-showcase.html), a ioslides presentation and Shiny apps developed as part of the weekly exercises in this subject at KU.

* [Population Genetics final exam](http://people.binf.ku.dk/rnq313/report.pdf). Working with SNP data from zebra species in the PLINK suite. The main topics focused on the genetic support for the distinction between plains subspecies and the position of the extinct _E. quagga quagga_ in the phylogenetic tree of zebras.

### Replicates of previously published scientific works ###

The projects below are just replicates of the published computational analysis carried out by other scientists. Thus, despite there might be minor differences, the bulk of the creation is *not* my own original work. Reference to the scientific article projects are based on may be found in the *references* section.

{% include portfolio.html %}

