<script>
$('.foldable').hide();
 
$('.foldable_toggle').each(function() {
	$(this).append('<span class="toggle">&nbsp;</span>');
	$(this).wrapInner('<a href="#"></a>');
});
 
$('.foldable_toggle a').click(function() {
	$(this).parent().next('.foldable').toggle('slow');
	$(this).toggleClass('unfolded');
	return false;
});
</script>

<div class="container">

    <div class="row">

      <div class="span4">
        <h4 class="foldable_toggle"><a href="http://people.binf.ku.dk/rnq313/BMS_1/html"> Microarray data analysis</a></h4>
        <div class="foldable">
        <p> Data for 5 conditions in mice lung tissue </p>
        <p><strong>What I learned:</strong></p>
          <ul>
          <li>Bioconductor and R packages: limma, affy</li>
          <li>Quality control of samples</li>
          <li>Graphics for results interpretation: scatter and volcano plots, Venn diagrams, heatmaps</li>
          <li>Microarray tech pros and cons </li>
          <li>GEO database</li>
          </ul>
          </div>
      </div><!--/span-->

      <div class="span4">
      <h4><a href="http://people.binf.ku.dk/rnq313/BMS_2/html"> RNA-seq data analysis</a></h4>
      <p> Data for 3 watering conditions in <em>A. thaliana</em> </p>
      <p><strong>What I learned:</strong></p>
        <ul>
          <li>Tuxedo protocol: bowtie, cufflinks, cuffmerge, cuffdiff and cummeRbund</li>
          <li>Working on a Sun Grid Engine cluster through the command line</li>
          <li>Gene set enrichment analysis (GSEA)</li>
          <li>Gene ontology (GO) </li> 
          <li>SRA database</li>
        </ul>
      </div><!--/span-->
    
    </div><!--/row-->

    <div class="row spacer25"></div>

    <div class="row">

      <div class="span4">
        <h4><a href="http://people.binf.ku.dk/rnq313/BMS_3/html"> Gene coexpression networks</a></h4>
        <p> Studying correlations in wine yeast gene expression </p>
        <p><strong>What I learned:</strong></p>
          <ul>
          <li>Network theory: scale-free and small-world networks</li>
          <li>Biological networks motifs: AR, DFL, IFFL and CFFL </li>
          <li>Cytoscape software and igraph package </li>
          <li>Correlation analysis </li>
          </ul>
      </div><!--/span-->

      <div class="span4">
      <h4><a href="http://people.binf.ku.dk/rnq313/BMS_4/html"> Cell physiology simulations </a></h4>
      <p> Simulation on Ca<sup>2+</sup> spontaneous peaks </p>
      <p><strong>What I learned:</strong></p>
        <ul>
          <li>Implement biological networks and motifs </li>
          <li>COPASI and Cell Designer software</li>
          <li>Effect of network motifs on its system's dynamics</li>
        </ul>
      </div><!--/span-->

    </div><!--/row-->

    <div class="row spacer25"></div>

    <div class="row">

      <div class="span4">
        <h4><a href="http://people.binf.ku.dk/rnq313/BAG_1/html"> RNA-seq and ChIP-seq integral analysis</a></h4>
        <p> Development of a workflow for data integral analysis. Gene Set Analysis was carried on subsets of interesting genes as found by RNA-seq and ChIP-seq </p>
        <p><strong>What I learned:</strong></p>
          <ul>
          <li>Shell scripting and blackboard messaging</li>
          <li>ChIP-seq analysis software: MACS and PeakAnalyzer </li>
          <li>Motif finders: HOMER</li>
          <li>KEGG Database </li>
          <li>ggplot2 and other "Hadley verse" R packages </li>
          </ul>
      </div><!--/span-->

      <div class="span4">
      <h4><a href="http://people.binf.ku.dk/rnq313/BAG_2/synteny.pdf"> Synteny study on Python </a></h4>
      <p> Analysis of <em>M. pneumoniae</em> and <em>M. genitalium</em> </p>
      <p><strong>What I learned:</strong></p>
        <ul>
          <li> Python scripting and Biopython module </li>
          Download script <a href="http://people.binf.ku.dk/rnq313/BAG_2/tarea_2.py"> here </a>
          <li> The time Linux utility</li>
        </ul>
      </div><!--/span-->

    </div><!--/row-->

    <div class="row spacer25"></div>


    <div class="row">

      <div class="span4">
        <h4><a href="http://people.binf.ku.dk/rnq313/BAG_3/html"> RNA-seq <em>de novo</em> analysis</a></h4>
        <p> <em>De novo</em> genome assembly on frog genome and differential expression analysis on 2 different development states </p>
        <p><strong>What I learned:</strong></p>
          <ul>
          <li> Genome assembly and assessment methods: Trinity </li>
          <li> Transdecoder </li>
          <li> KAAS </li>
          </ul>
      </div><!--/span-->

      <div class="span4">
      <h4><a href="http://people.binf.ku.dk/rnq313/BAG_4/tarea4.pdf"> Variant call analysis </a></h4>
      <p> Discovering differences between an arbitrary <em>A. thaliana</em> genome and a reference genome </p>
      <p><strong>What I learned:</strong></p>
        <ul>
          <li> bwa </li>
          <li> GATK software suite </li>
          Download shell script <a href="http://people.binf.ku.dk/rnq313/BAG_4/GWAS.sh"> here </a>
        </ul>
      </div><!--/span-->

    </div><!--/row-->

    <div class="row spacer25"></div>



</div>


